tests/kafkatest/services/console_consumer.py

```diff
@@ -184,9 +183,12 @@ def start_cmd(self, node):
 
         # LoggingMessageFormatter was introduced after 0.9
         if node.version > LATEST_0_9:
-            cmd+=" --formatter kafka.tools.LoggingMessageFormatter"
+            cmd += " --formatter kafka.tools.LoggingMessageFormatter"
+
+        # enable systest events is only available in 0.10.0 and later
+        if node.version >= V_0_10_0_0:
```

Comment:
I don't think it's an issue for this patch, but I'm realizing that we may not have clearly defined when we bump version numbers pre-release. Here, we're clearly saying that we've already marked `V_0_10_0_0`, but that's actually a bit weird given that we're still on a dev version, right? Or does this not pass until we actually remove the `dev0` suffix?